onfido-java/src/main/java/com/onfido/TaskManager.java

```diff
@@ -0,0 +1,32 @@
+package com.onfido;
+
+import com.onfido.api.ApiJson;
+import com.onfido.api.Config;
+import com.onfido.api.ResourceManager;
+import com.onfido.exceptions.OnfidoException;
+import com.onfido.models.Task;
+import com.onfido.models.TaskLight;
+import okhttp3.OkHttpClient;
+
+import java.util.List;
+
+public class TaskManager extends ResourceManager {
+    private ApiJson<TaskLight> taskLightParser = new ApiJson<>(TaskLight.class);
+    private ApiJson<Task> taskParser = new ApiJson<>(Task.class);
+
+    protected TaskManager(Config config, OkHttpClient client) {
+        super("workflow_runs/", config, client);
+    }
+
+    public TaskManager(String basePath, Config config, OkHttpClient client) {
+        super(basePath, config, client);
+    }
+
+    public List<TaskLight> list(String workflowRunId) throws OnfidoException {
+        return taskLightParser.parseList(get(workflowRunId + "/tasks"));
+    }
+
+    public Task find(String workflowRunId, String taskId) throws OnfidoException {
+        return taskParser.parse(get(workflowRunId + "/tasks/" + taskId));
+    }
+}
```

Comment:
The class is missing the definition for `complete` [action](https://documentation.onfido.com/#complete-task).